mindbender/plugins/collect_project_editorial_info.py

```diff
@@ -0,0 +1,36 @@
+# collect_project_editorial_info.py
+import pyblish.api
+
+
+class CollectMindbenderEditInfo(pyblish.api.ContextPlugin):
+    """Store projects editorial info at the time of publish"""
+
+    label = "Collect Project Edit Info"
+    order = pyblish.api.CollectorOrder
+    hosts = ["maya"]
+    families = [
+        # More will come
```

Comment:
No need for notes like these.

If you have an important but *minor* message to your future self or other developer, you may use `# TODO(yourname): Message here` or `# NOTE(yourname): Message here`. But they are to be used sparingly. For major notes, make an issue on GitHub.